### DISCUSSION TITLE
REST API howto

### DISCUSSION THREAD

**Discussion by syxys1** [illacloud/illa-builder#3543] [Q&A] [answered]
Congratulation on your product.  I have work on Retool and I feel right at home.  I am trying to to query Clockify REST API.  This is something I succeeded in Retool.  The problem in Illa is when buiding the REST api request, I need to attach a JSON body but this is not an option right now.  Do you have a workaround or a target date to have this implemented?

**Comment by owenlongbo** [comment, marked as answer]
![image](https://github.com/illacloud/illa-builder/assets/7929686/8d135b21-21c8-42df-8e0f-b8add524defa)
Is Right?

**Reply by syxys1** [reply]
Thank you!
